config/environments/production.rb

```diff
@@ -59,9 +57,9 @@
   # Use a different cache store in production.
   # config.cache_store = :mem_cache_store
 
-  # Use a real queuing backend for Active Job (and separate queues per environment)
+  # Use a real queuing backend for Active Job (and separate queues per environment).
   # config.active_job.queue_adapter     = :resque
-  # config.active_job.queue_name_prefix = "CodeLabs_#{Rails.env}"
+  # config.active_job.queue_name_prefix = 'code_labs_production'
```

Comment:
Is this OK to use a hardcoded prefix instead of generating from ENV?

Reply:
> config/environments/production.rb

It's always production, it's commented out, and that's the default from rails 6. So yes, it's OK.